### PR TITLE
SAN-136: Update versions of vulnerable packages

### PR DESCRIPTION
**Pull request by acallaghan-ch**
Update versions of vulnerable packages

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,16 +27,15 @@
         <api-sdk-java.version>6.0.19</api-sdk-java.version>
 
         <thymeleaf-layout-dialect.version>3.3.0</thymeleaf-layout-dialect.version>
-        <spring-boot-dependencies.version>3.3.3</spring-boot-dependencies.version>
-        <spring-boot-maven-plugin.version>3.3.3</spring-boot-maven-plugin.version>
+        <spring-boot-dependencies.version>3.3.5</spring-boot-dependencies.version>
```
Comment (rbull-ch) [inline pom.xml:30]: - Please can you try 3.3.6 like in previous comment?
- You'll need to remove the exclude for the `spring-web` under `spring-boot-starter-web`

https://github.com/companieshouse/penalty-payment-web/blob/5cdf6cbaeb2d5a5696a1c2d544f98b6c9c7a896d/pom.xml#L77-L82

Comment (acallaghan-ch) [reply pom.xml:30]: 3.3.5 is the latest available in the Maven Repo:
https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies
The other commit has 3.3.4 (updated from 3.3.3)
I will remove the exclusion and the explicit dependency.

Comment (rbull-ch) [reply pom.xml:30]: It worked fine for me locally - it comes from artifactory (https://repository.aws.chdev.org/artifactory/webapp/#/home)

Comment (acallaghan-ch) [reply pom.xml:30]: Dead on, can bump it up to 3.3.6

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by dgordon-ch — APPROVED** [review]

**Pull request merged**